resources/definitions/lotmaxx_sc10.def.json

```diff
@@ -0,0 +1,64 @@
+{
+    "name": "Lotmaxx SC-10",
+    "version": 2,
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "lotmaxx.com",
+        "manufacturer": "Lotmaxx",
+        "file_formats": "text/x-gcode",
+        "platform": "lotmaxx_sc_10_20_platform.stl",
+        "has_materials": true,
+        "machine_extruder_trains":
+        {
+            "0": "lotmaxx_sc10_extruder_0"
+        }
+    },
+    "overrides": {
+        "machine_name": { "default_value": "Lotmaxx SC-10" },
+        "machine_width": { "default_value": 235 },
+        "machine_depth": { "default_value": 235 },
+        "machine_height": { "default_value": 320 },
+        "machine_head_polygon": { "default_value": [[-32,11],[-32,-32],[28,-32],[28,11]] },
```

Comment:
The test failed because of this setting. There's no longer `machine_head_polygon` as a setting. You can use `machine_head_with_fans_polygon` instead.

Log from the tests:
```
./tests/Settings/TestDefinitionContainer.py:59: AssertionError
----------------------------- Captured stdout call -----------------------------
[MainThread] UM.Settings.DefinitionContainer._preprocessParsedJson [270]: Unable to override setting machine_head_polygon
=============================== warnings summary ===============================
```